### PR TITLE
A page for the new Quantum Computing WG

### PR DESCRIPTION
**Pull request by ric-bianchi**
plus, some small changes to the Visualization page.

### CODE REVIEW

File: _workinggroups/quantumcomputing.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# The HSF Quantum Computing working group
+
+## Introduction
+
+The Quantum Computing Working Group fosters collaboration among the HEP experiments and institutes to exploit the possibilities offered by quantum computing.
+
+Quantum computing is a very active research field and prototype computers, simulators, and toolkits are now available. The participants in the working group share experiences, ideas, know-how, demonstrators, code snippets, and documents with the aim of fostering common research and development in this field, for the benefit of the HEP community.
+
+
+[Mandate](/organization/working-group-mandates.html).
```
Comment (graeme-a-stewart) [inline _workinggroups/quantumcomputing.md]: It seems a bit off to reference the mandate page, which was written more specifically with Sim, Reco and Analysis groups in mind. I think the activities is generally good, and we should probably factorise that for all WGs, but the specific page referenced is a bit too CWPish, where we didn't have QC at the time.

File: _workinggroups/quantumcomputing.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Kick-starters: Riccardo Maria BIANCHI (ATLAS), Sebastien BINET (LSST, ALICE, LPC), Riccardo DI SIPIO (ATLAS), Lukas HEINRICH (ATLAS), Maurizio PIERINI (CMS)
+
+----
+
+## Repository
+
+You can find code snippets and demonstrators in the working group's GitHub repository: <https://github.com/HSF/quantum-computing>. All code, unless differently specified, is released under the Apache 2.0 license.
```
Comment (graeme-a-stewart) [inline _workinggroups/quantumcomputing.md]: I think the Apache license is not correct here - that should be in the code repository itself.

File: _workinggroups/visualization.md [hunk 3]
```diff
@@ -63,12 +67,18 @@ Topics for which we will have to interact with other WGs:
...
+
+* HSG "Visualization" Community White Paper: <https://arxiv.org/abs/1811.10309>
 
 ----
 
 ## Meetings
 
+### HSF Mini-workshop on GUI, graphic libraries, interactive visualization, CERN, 7 April 2017
+
+* Agenda: https://indico.cern.ch/event/628675/
```
Comment (graeme-a-stewart) [inline _workinggroups/visualization.md]: You could make that a link.

File: _workinggroups/visualization.md [hunk 4]
```diff
@@ -63,12 +67,18 @@ Topics for which we will have to interact with other WGs:
...
+
 ### HSF Visualization Workshop, CERN, 28-30 March 2017
 
 * Agenda: https://indico.cern.ch/event/617054/
```
Comment (graeme-a-stewart) [inline _workinggroups/visualization.md]: Link.

**Comment by graeme-a-stewart** [comment]
Hi @ric-bianchi - little ping on this one, would be good to get it done.

File: _workinggroups/quantumcomputing.md (hunk 2 above)
Comment (ric-bianchi) [reply _workinggroups/quantumcomputing.md]: Hi Graeme,
Thanks for your comments and suggestions.

I have the LICENSE file in the QC repo, already: https://github.com/HSF/quantum-computing 

Sorry, but I didn't really understand your comment: should I remove the mention to the license here and leave the license in the repository only?

Best,
--    Ric.

File: _workinggroups/visualization.md (hunk 3 above)
Comment (ric-bianchi) [reply _workinggroups/visualization.md]: Done.

File: _workinggroups/visualization.md (hunk 4 above)
Comment (ric-bianchi) [reply _workinggroups/visualization.md]: Done.

File: _workinggroups/quantumcomputing.md (hunk 2 above)
Comment (graeme-a-stewart) [reply _workinggroups/quantumcomputing.md]: Yes, it's not relevant to mention it here. Keep the it in the repo where it belongs.

Comment (ric-bianchi) [reply _workinggroups/quantumcomputing.md]: OK, thanks.
Done.

**Comment by ric-bianchi** [comment]
> 
> 
> @ric-bianchi - thanks a lot for this. I have a few small observations on the change.
> 
> I also noticed that the `layout` of the visualisation page is wrong and inconsistent with the other working group pages:
> 
>     * The `layout` should be `plain`
> 
>     * The `redirect_from` is wrong and should be `/activities/visualization.html` (`git blame` says it's my mistake, sorry!, but could you fix it in this PR?)
> 
> 
> Thanks!

Layout and redirect_from have been fixed.

Should I add a redirect_from link to QC as well? But I see that's not listed in the Activities/WGs on the site yet.

File: _workinggroups/visualization.md [hunk 5]
```diff
@@ -1,50 +1,54 @@
...
+layout: plain
+redirect_from: /activities/visualization.html
 ---
 
 # Visualization
 
-The Visualization Working Group (WG) gathers experts, developers, users and all people from the HEP community interested in interactive data visualization and event displays. The WG has been started in October 2016 and it is convened by Riccardo Maria Bianchi (*University of Pittsburgh*).
+The Visualization Working Group (WG) gathers experts, develeopers, users and all people from the HEP community interested in interactive data visualization and event displays. 
+
+The WG has been started in October 2016 and it is convened by Riccardo Maria Bianchi (*University of Pittsburgh, US*, ATLAS Experiment).
```
Comment (graeme-a-stewart) [inline _workinggroups/visualization.md]: You should put some contact information in here - which should be a way to contact the convenor (i.e., you!) and a way to contact the group (I suppose there's a mailing list? A few of the groups renamed their mailing lists to remove the "cwp" part, but retain the established contacts).

Comment (graeme-a-stewart) [reply _workinggroups/visualization.md]: Right, so checking, what I suggest is to rename "hsf-cwp-visualization" -> "hsf-visualization-wg"

File: _workinggroups/quantumcomputing.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+# The HSF Quantum Computing working group
+
+## Introduction
+
+The Quantum Computing Working Group fosters collaboration among the HEP experiments and institutes to exploit the possibilities offered by quantum computing.
+
+Quantum computing is a very active research field and prototype computers, simulators, and toolkits are now available. The participants in the working group share experiences, ideas, know-how, demonstrators, code snippets, and documents with the aim of fostering common research and development in this field, for the benefit of the HEP community.
+
+Kick-starters: *Riccardo Maria BIANCHI* (ATLAS), *Sebastien BINET* (LSST, ALICE, LPC), *Riccardo DI SIPIO* (ATLAS), *Lukas HEINRICH* (ATLAS), *Maurizio PIERINI* (CMS)
```
Comment (graeme-a-stewart) [inline _workinggroups/quantumcomputing.md]: Contact info is needed - one way to contact the convenors and one way to contact the group. I think the group email list exists, right? For the convenors it's probably best that you setup a Google group with the 5 of you on it.

**Comment by graeme-a-stewart** [comment]
Hi @ric-bianchi 

Thanks for updating - looking better now. There are still a few things that I think can be improved though - mostly contact info.

You don't need to add any redirect for QC. In the past the WGs were listed under "activities", so when the pages were moved we put in a redirect. But for new pages it's not necessary.

**Comment by ric-bianchi** [comment]
> 
> 
> Hi @ric-bianchi
> 
> Thanks for updating - looking better now. There are still a few things that I think can be improved though - mostly contact info.
> 
> You don't need to add any redirect for QC. In the past the WGs were listed under "activities", so when the pages were moved we put in a redirect. But for new pages it's not necessary.

OK, Thanks Graeme. Let me know if I can help further.

Have a good day,
--   Ric.

File: _workinggroups/visualization.md (hunk 5 above)
Comment (ric-bianchi) [reply _workinggroups/visualization.md]: Sorry, Just saw this one! 
I'm checking that now.

Comment (ric-bianchi) [reply _workinggroups/visualization.md]: OK, done. Added to both Viz and QC.

Comment (ric-bianchi) [reply _workinggroups/visualization.md]: and I renamed the Viz mailing list

File: _workinggroups/quantumcomputing.md (hunk 6 above)
Comment (ric-bianchi) [reply _workinggroups/quantumcomputing.md]: Done, and added to the QC page.

File: _workinggroups/visualization.md [hunk 7]
```diff
@@ -1,50 +1,62 @@
 ---
 title: "Visualization"
 author: "Riccardo Maria Bianchi"
-layout: default
-redirect_from: /activities/training.html
+layout: plain
+redirect_from: /activities/visualization.html
 ---
 
 # Visualization
```
[the remark this answers is not in the document]
Comment (ric-bianchi) [reply _workinggroups/visualization.md]: Done.

[the remark this answers is not in the document]
Comment (ric-bianchi) [reply _workinggroups/visualization.md]: Also in QC.

**Comment by ric-bianchi** [comment]
The CI checks are failing on all my commits. Is that expected?

File: _workinggroups/quantumcomputing.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+## Contacts
+
+- Convenors: [hsf-quantumcomputing-wg-convenors@googlegroups.com](mailto:hsf-quantumcomputing-wg-convenors@googlegroups.com)
+- Working group's mailing list: [hsf-quantumcomputing-wg@googlegroups.com](hsf-quantumcomputing-wg@googlegroups.com)
```
Comment (graeme-a-stewart) [inline _workinggroups/quantumcomputing.md]: Ah! Last mistake - this should be a `mailto:` link.

Comment (ric-bianchi) [reply _workinggroups/quantumcomputing.md]: Yes! Good catch!
Thanks!

Now, fixed.

**Pull request merged**